### PR TITLE
LPS-173573 Get the correct value for Attachment Fields

### PR DESCRIPTION
**Pull request by guilhermedcamacho**
Related issue: https://issues.liferay.com/browse/LPS-173573

### CODE REVIEW

**Comment by marcelabc** [comment]
I just wanted to let you know that I edited the description to include the "Related Issue" (with the Jira ticket). Please always add this information in your following reviews if you can :smile:

**Comment by marcelabc** [comment]
Just started reviewing :)

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/renderer/ObjectEntryRowInfoItemRenderer.java [hunk 1]
```diff
@@ -198,6 +202,28 @@ else if (Validator.isNotNull(
 
 						return StringPool.BLANK;
 					}
+					else if (Objects.equals(
```
Comment (marcelabc) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/renderer/ObjectEntryRowInfoItemRenderer.java:205]: Could you reorder this condition? 
I believe this condition must be before this one:
`Objects.equals(objectField.getDBType(),ObjectFieldConstants.DB_TYPE_DATE)`

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by brianchandotcom** [comment]
@marcelabc @guilhermedcamacho see 4444188757b32588356b95ec460f7ebadae6379b

If order MATTERS, I'd like to add a line break.

So

if (x) {
}

if (y) {
}
else if (z) {
}

So x HAS to go before y and z, but y and z don't matter.

Can you guys send me a follow up PR to do this?

Thx.

**Comment by guilhermedcamacho** [comment]
> @marcelabc @guilhermedcamacho see [4444188](https://github.com/liferay-objects/liferay-portal/commit/4444188757b32588356b95ec460f7ebadae6379b)
> 
> If order MATTERS, I'd like to add a line break.
> 
> So
> 
> if (x) { }
> 
> if (y) { } else if (z) { }
> 
> So x HAS to go before y and z, but y and z don't matter.
> 
> Can you guys send me a follow up PR to do this?
> 
> Thx.

I will do that, thanks!

**Comment by guilhermedcamacho** [comment]
I checked the CI errors, they are not related. I will send the follow up PR this week.

**Pull request closed without merging**